From 56d0e6bf63f40635b25cf9edd9b989bf39b0eb0f Mon Sep 17 00:00:00 2001
From: Vince Chang <vince_chang@aspeedtech.com>
Date: Mon, 29 Jun 2026 14:30:12 +0800
Subject: [PATCH] Use aspeed's novnc fork

aspeed's novnc fork supports aspeed proprietary jpg.
Vite handles the fork's worker and WebAssembly assets during build.

Upstream-Status: Inappropriate [ASPEED specific]
Signed-off-by: Jammy Huang <jammy_huang@aspeedtech.com>
Signed-off-by: Vince Chang <vince_chang@aspeedtech.com>
---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 7bcbd69..428b0a0 100644
--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
     },
     "dependencies": {
         "@carbon/icons-vue": "10.49.1",
-        "@novnc/novnc": "1.3.0",
+        "@novnc/novnc": "git+https://github.com/AspeedTech-BMC/noVNC.git#346ec35",
         "@tanstack/vue-query": "5.92.9",
         "@vue/vue3-jest": "27.0.0",
         "@vuelidate/core": "2.0.3",
-- 
2.43.0
